tests/cases/fourslash/completionsClassMemberImportTypeNodeParameter2.ts

```diff
@@ -18,7 +18,7 @@ verify.completions({
   includes: [
     {
       name: "method",
-      insertText: `method(param: import("./doesntexist.js").Foo);`,
+      insertText: `method(param: any);`,
```

Comment:
This seems like a case where we would have allowed unresolved names before, so maybe we also need to vet everyone who passed NoTruncation to see if they need AllowUnresolvedNames?